### PR TITLE
Shaman demos to string

### PR DESCRIPTION
**Pull request by asturur**
replace #78 and #79

Currently investigating why the poly demo indeed doesn't work.

### CODE REVIEW

**Comment by asturur** [comment]
plan is:
i copy the relevant code in the demo,
i debug, i either understand or fix, and create an interaction test e2e.

**Comment by asturur** [comment]
closes https://github.com/fabricjs/fabricjs.github.io/pull/78 and https://github.com/fabricjs/fabricjs.github.io/pull/79

**Comment by ShaMan123** [comment]
Why did you close my PRs?
You should leave them open until we move to js.
Closing them will make my work nullified and lost.
Keeping them open means we can reuse them in time be.

File: src/content/demo/poly-controls/index.md
[the remark this answers is not in the document]
Comment (asturur) [reply src/content/demo/poly-controls/index.md:1]: there is a pr to fix createPolyControls in fabricjs. The demo still works it shows you that can be done. We can simpli add that the the logic has been wrapped in a util and expand the explanation of how it works

[the remark this answers is not in the document]
Comment (asturur) [reply src/content/demo/poly-controls/index.md:1]: or we could also not use the util. But the code of the demo would be a lot

[the remark this answers is not in the document]
Comment (ShaMan123) [reply src/content/demo/poly-controls/index.md:1]: we should use it
think of a dev coming to the demo and thinking they need to recreate it.
I would drop all this and add a link to the impl in fabric
that is the best IMO cause it keeps the demo clean while giving an option for a deep dive if the dev choose to

**Pull request merged**